### ISSUE TITLE
Warning color fails WCAG accessibility test in light mode

### ISSUE THREAD

**Issue by kaishin** [NSHipster/nshipster.com#59]
Currently the score is a measly `1.44`.

Something more along the lines of `#b28106` passes the test with a ratio of `3.04`.

<img width="898" alt="screenshot-2020-04-13" src="https://user-images.githubusercontent.com/519433/79120997-8c422b80-7d94-11ea-97d7-3bd7eb4bf82d.png">

**Comment by mattt** [comment]
Yikes! You're right, that's far too light. Sorry to introduce such a glaring A11y issue in your great article. #b28106 looks good (3.11 against #f2f2f7 according to https://webaim.org/resources/contrastchecker/; not sure about the discrepancy). I'll patch that in now.
